README.md

```diff
@@ -101,6 +110,8 @@ _NOTE:_ Node.js and npm both need to be installed on the host machine, not in th
     npm install
 3. Run `$(npm bin)/gulp` in the root of the project to generate the JS and CSS files.
 
+You will need to ask a co-worker for the Fort Awesome npm config setting. You will enter an `npm config set` command so that this config value is available during the setup script.
```

Comment:
Fort -> Font

Reply:
Technically, the repo is called fort-awesome. Tripped me up the first time too.

Reply:
@paulrobinson I wouldn't feel too bad. It caught me out first time I saw it as well. TBH it's a terrible name for the repo.....